src/SDKs/CognitiveServices/CognitiveServices.Tests/Tests/CognitiveServicesAccountTests.cs

```diff
@@ -156,14 +155,14 @@ public void CognitiveServicesAccountListByResourceGroupTest()
                 // Create resource group
                 var rgname = CognitiveServicesManagementTestUtilities.CreateResourceGroup(resourcesClient);
 
-                var accounts = cognitiveServicesMgmtClient.CognitiveServicesAccounts.ListByResourceGroup(rgname);
+                var accounts = cognitiveServicesMgmtClient.Accounts.ListByResourceGroup(rgname);
```

Comment:
@felixwa the latest version is here: https://www.nuget.org/packages/Microsoft.Azure.Management.ResourceManager/1.6.0-preview